### ISSUE TITLE
add support for including files that aren't namespaced

### ISSUE THREAD

**Issue by yoshuawuyts** [sheetify/sheetify#35]
From #32 

Scoping styles to components is ideal as a transform, but sometimes global files must be included too (such as `reset.css`). These files shouldn't be namespaced, but should be included in the resulting bundle.

- What syntax do we want for this?
- Should these files perhaps only be prepended during an [exorcise](https://github.com/sheetify/sheetify/issues/34) step?
- any other things to consider?

**Comment by ahdinosaur** [comment]
strawman proposal:

```js
var sheetify = require('sheetify')

sheetify('./reset.css', { global: true })
```

where everything is the same except it doesn't run the css through [`postcss-prefix`](https://github.com/sheetify/postcss-prefix) and returns `null`.

**Comment by yoshuawuyts** [comment]
yeah, this looks good to me! Would probably also want to support `require` calls with this syntax (#36):
```js
var sheetify = require('sheetify')

sheetify('css-wipe', { global: true })
sheetify('css-scale', { global: true })
sheetify('css-type-base', { global: true })
sheetify('./index.css', { global: true })
```

**Comment by ahdinosaur** [comment]
hehe, i was thinking the same. :smile:

but then an open question is how do we distinguish between a module path and valid css? or is it obvious enough?

**Comment by yoshuawuyts** [comment]
Closed by #46
